elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -62,7 +76,31 @@
 
     protected QueryEngine.Transaction transaction;
 
-    public static void init() {
+    // Standard set of test queries used in dialect tests
+    protected enum TestQueryName {
+        WHERE_METRICS_ONLY,
+        WHERE_DIMS_ONLY,
+        WHERE_METRICS_AND_DIMS,
+        WHERE_METRICS_OR_DIMS,
+        WHERE_METRICS_AGGREGATION,
+        HAVING_METRICS_ONLY,
+        HAVING_DIMS_ONLY,
+        HAVING_METRICS_AND_DIMS,
+        HAVING_METRICS_OR_DIMS,
+        PAGINATION_TOTAL,
+        SORT_METRIC_ASC,
+        SORT_METRIC_DESC,
+        SORT_DIM_DESC,
+        SORT_METRIC_AND_DIM_DESC,
+        SUBQUERY,
```

Comment:
It would be good to break out most of these feature tests by table and by subquery.  By features, I mean:
WHERE
HAVING
TIME Dimensions
PAGINATION
SORTING

Reply:
Is your ask to create two versions of the existing tests - 1 for table and 1 for subquery? Or just to update the names to clearly show whether they're table or subquery tests?

Reply:
The former - I think we should test most _features_ for both tables and subqueries.